packages/dso-toolkit/src/components/card/card.mixins.scss

```diff
@@ -13,27 +15,24 @@
   }
 }
 
-%card-grid-addon {
-  grid-row: span 2;
-}
-
 @mixin base() {
   padding-block: units.$u2 units.$u1;
   padding-inline: units.$u2;
   inline-size: 100%;
 
-  &:hover {
+  &:has(.dso-card-heading > a:hover) {
     background-color: colors.$grijs-5;
-    cursor: pointer;
 
-    .dso-card-heading > a {
+    .dso-card-heading > a:hover {
+      color: card-variables.$heading-anchor-hover-color;
       text-decoration: underline;
     }
   }
+  &:focus-within {
+    background-color: colors.$grijs-5;
+  }
 
-  &:not([href]):not([clickable]),
-  &:not([href])[clickable="false"],
-  &.dso-not-clickable {
+  &:not([href]) {
```

Comment:
Is het nodig om hier styling op toe te passen?

Reply:
Denk het niet;`href` is immers altijd aanwezig.

Ik heb het dus weggehaald